src/sentry/integrations/mixins/issues.py

```diff
@@ -195,8 +195,10 @@ def store_issue_last_defaults(self, project: Project, user: APIUser, data):
         if user_persisted_fields:
             user_defaults = {k: v for k, v in data.items() if k in user_persisted_fields}
             user_option_key = dict(key="issue:defaults", project_id=project.id)
```

Comment:
Very minor nit: how about typing this now that we can?  `user_option_key: UserOptionFilterArgs`.  Catches problems in the future if we make a change.